tests/src/test/scala/cats/collections/BitSetTest.scala

```diff
@@ -222,4 +223,109 @@ object BitSetTest extends Properties("BitSet") {
       val rhs = x.reverseIterator.toList
       (lhs == rhs) :| s"$lhs == $rhs"
     }
+
+  property("((x ^ y) ^ z) = (x ^ (y ^ z))") =
+    forAll { (x: BitSet, y: BitSet, z: BitSet) =>
+      val lhs = ((x ^ y) ^ z)
+      val rhs = (x ^ (y ^ z))
+      (lhs == rhs) :| s"$lhs == $rhs"
+    }
+
+  property("(x ^ y) = (y ^ x)") =
+    forAll { (x: BitSet, y: BitSet) =>
+      val lhs = (x ^ y)
+      val rhs = (y ^ x)
+      (lhs == rhs) :| s"$lhs == $rhs"
+    }
+
+  property("(x ^ x) = 0") =
+    forAll { (x: BitSet) =>
+      val lhs = (x ^ x)
+      val rhs = BitSet.empty
+      (lhs == rhs) :| s"$lhs == $rhs"
+    }
+
+  property("(x ^ 0) = x") =
+    forAll { (x: BitSet) =>
+      val lhs = (x ^ BitSet.empty)
+      val rhs = x
+      (lhs == rhs) :| s"$lhs == $rhs"
+    }
+
+  property("(x ^ y) = ((x -- (x & y)) | (y -- (x & y)))") =
```

Comment:
Can we directly test `(x ^ y)(z) == x(z) ^ y(z)` ?

Reply:
Done. Also added a similar property for `(x -- y)`.